ibmdbpy/tests/test_base_connexion.py

```diff
@@ -101,6 +101,28 @@ def test_idadb_as_idadataframe(self, idadb, df):
         assert(ida.shape == df.shape)
         idadb.drop_table("TEST_AS_IDADF_18729493954_23849590")
 
+    # test schema support
```

Comment:
Python docstrings are usually on the line after the function declaration (see [PEP-257](https://www.python.org/dev/peps/pep-0257/)):

```python
    def test_idadb_as_idadataframe_with_schema(self, idadb, df):
        """Test schema support"""
        [...]
```

Reply:
This is not a docstring, but an internal comment that should not appear in any documentation. So I did not change that.